src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -58,11 +58,18 @@ const getStyleProps = (
 };
 
 const defaultGraphColumnsSettings: GKGraphColumnsSettings = {
-	commitAuthorZone: { width: 110 },
-	commitDateTimeZone: { width: 130 },
-	commitMessageZone: { width: 130 },
-	commitZone: { width: 170 },
-	refZone: { width: 150 },
+	commitAuthorZone: { width: 110, isHidden: false },
+	commitDateTimeZone: { width: 130, isHidden: false },
+	commitMessageZone: { width: 130, isHidden: false },
+	commitZone: { width: 170, isHidden: false },
+	refZone: { width: 150, isHidden: false },
+};
+
+const getGraphColConfigModel = (columnSettings: GKGraphColumnSetting): GraphColumnConfig => {
+	return {
+		width: columnSettings.width,
+		isHidden: columnSettings.isHidden,
+	};
```

Comment:
Does `GKGraphColumnSetting` have more properties than `width` and `isHidden`? If not, this function is likely not needed.

Reply:
Done in commit: `Refactoring: removed "getGraphColConfigModel" function`